_source/_posts/2022-07-20-express-prisma-tailwind.md

```diff
@@ -0,0 +1,665 @@
+---
+layout: blog_post
+title: "How to Build an Express Application using Prisma"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "Learn how to build a workout tracker application using Express and Prisma and secure it using Okta."
+tags: [prisma, node, express, passport]
+tweets:
+- "Learn how to build a workout tracker application using Express and Prisma and secure it using Okta ⚡️"
+image: blog/express-prisma-and-tailwindcss/social.jpg
+github: https://github.com/oktadev/okta-express-prisma-tailwind-example
+type: conversion
+---
+
+[Express](https://expressjs.com/) is one of the most popular web frameworks available today.  Understanding the tools available when building any web application is critical to being a good developer.
+
+[Prisma](https://www.prisma.io/) is an ORM for Node.js using TypeScript.  It integrates with many of the most popular databases today, including MySQL, SQL Server, SQLite, and MongoDB.   Prisma emphasizes a human-readable schema with a type-safe database client.  Prisma includes other features such as migrations, seed data, and a virtual database browser.   
```

Comment:
```suggestion
[Prisma](https://www.prisma.io/) is an ORM (object–relational mapping) tool for Node.js using TypeScript.  The software integrates with many of the most popular databases today, including MySQL, SQL Server, SQLite, and MongoDB, and emphasizes a human-readable schema with a type-safe database client.  Prisma also includes other features such as migrations, seed data, and a virtual database browser.   
```
Avoids starting a series of sentences with the same first word. Spells out the first use of an acronym that may be unfamiliar to some readers.